composer.json

```diff
@@ -28,8 +28,17 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
-        "codeclimate/php-test-reporter": "^0.4",
+        "phpunit/phpunit": "^9",
         "squizlabs/php_codesniffer": "^3.2"
+    },
+    "config": {
+        "lock": false
+    },
+    "scripts": {
+        "tests": "phpunit",
+        "ci": [
+            "@composer validate --strict",
+            "@build"
```

Comment:
v buildu to pise
> Run composer ci
./composer.json is valid
You made a reference to a non-existent script @build

Jakej byl zamer s tim `@build` - pustit phpunit testy? Potom by tam mel byt `@tests` ;-)

Reply:
:D opraveno, sorry

Reply:
bylo by fajn, kdyby to ten skript skoncil, zajimavy ze to jen vypise warning a frci to dal...

Reply:
To je ale vlastnost composeru když je tam ta stage není nebo je prázdná tak vrátí status code 0